Manito/Manito/Screens/Letter/LetterImageViewController.swift

```diff
@@ -8,158 +8,88 @@
-            
-            PHPhotoLibrary.shared().performChanges({
-                PHAssetChangeRequest.creationRequestForAsset(from: image)
-            }) { (success, error) in
-                DispatchQueue.main.async {
-                    if success {
-                        self?.makeAlert(title: TextLiteral.letterImageViewControllerSuccessTitle,
-                                        message: TextLiteral.letterImageViewControllerSuccessMessage)
-                    } else if let error = error {
-                        Logger.debugDescription(error)
-                        self?.makeAlert(title: TextLiteral.letterImageViewControllerErrorTitle,
-                                        message: TextLiteral.letterImageViewControllerErrorMessage)
-                    }
-                }
+                                message: error.errorDescription)
             }
         }
-        self.downloadButton.addAction(downloadAction, for: .touchUpInside)
-    }
-    
-    // MARK: - selector
-    
-    @objc
-    private func didTapCloseButton() {
-        self.dismiss(animated: true)
     }
-    
-    @objc
-    private func didPinchImage(_ pinch: UIPinchGestureRecognizer) {
-        self.imageView.transform = self.imageView.transform.scaledBy(x: pinch.scale, y: pinch.scale)
-        pinch.scale = 1
-    }
-}
 
-extension LetterImageViewController: UIScrollViewDelegate {
-    func viewForZooming(in scrollView: UIScrollView) -> UIView? {
-         return self.imageView
-     }
-    
-    func scrollViewDidZoom(_ scrollView: UIScrollView) {
-        if scrollView.zoomScale > 1 {
-            guard let image = self.imageView.image else { return }
-            guard let zoomView = self.viewForZooming(in: scrollView) else { return }
-            
-            let widthRatio = zoomView.frame.width / image.size.width
-            let heightRatio = zoomView.frame.height / image.size.height
-            let ratio = widthRatio < heightRatio ? widthRatio : heightRatio
-            
-            let newWidth = image.size.width * ratio
-            let newHeight = image.size.height * ratio
-            
-            let left = 0.5 * (newWidth * scrollView.zoomScale > zoomView.frame.width ?
-                              (newWidth - zoomView.frame.width) : (scrollView.frame.width - scrollView.contentSize.width))
-            let top = 0.5 * (newHeight * scrollView.zoomScale > zoomView.frame.height ? (newHeight - zoomView.frame.height) : (scrollView.frame.height - scrollView.contentSize.height))
-            
-            scrollView.contentInset = UIEdgeInsets(top: top.rounded(), left: left.rounded(), bottom: top.rounded(), right: left.rounded())
-        } else {
-            scrollView.contentInset = .zero
+    private func uploadImage(for image: UIImage?, completionHandler: @escaping ((Result<(title: String, message: String), LetterImageError>) -> ())) {
+        guard let image = image else { completionHandler(.failure(.invalidImage)); return }
```

Comment:
p5;

```suggestion
        guard let image else { completionHandler(.failure(.invalidImage)); return }
```
위와 같이 수정할 수 있긴 합니다 !
p5입니다 p5

Reply:
반영했습니다!
[d0c53e7](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/386/commits/d0c53e732e2681e47c00e4ae3a6ad750882673b4)